src/common/Common.h

```diff
@@ -48,6 +48,8 @@
 #define stricmp strcasecmp
 #endif
 
+inline unsigned long atoul(char const* str) { return strtoul(str, nullptr, 10); }
```

Comment:
No need this.
https://github.com/azerothcore/azerothcore-wotlk/blob/master/src/common/Utilities/StringConvert.h